### PR TITLE
fix : create facility dashhboard

### PR DESCRIPTION
**Pull request by jabahum**
- create a facility dashboard

### CODE REVIEW

File: packages/esm-ugandaemr-app/src/createDashboardLink.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React, { useMemo } from 'react';
+import { ConfigurableLink } from '@openmrs/esm-framework';
+import { BrowserRouter, useLocation } from 'react-router-dom';
+
+export interface DashboardLinkConfig {
+  name: string;
+  title: string;
+}
+
+function DashboardExtension({ dashboardLinkConfig }: { dashboardLinkConfig: DashboardLinkConfig }) {
```
Comment (pirupius) [inline packages/esm-ugandaemr-app/src/createDashboardLink.tsx:10]: i don't think u need this. There slots available and u could just attach them the same way i add links to the patient chart.

File: packages/esm-ugandaemr-app/src/dashboard.meta.tsx [hunk 2]
```diff
@@ -4,3 +4,19 @@
 //   config: { columns: 1, type: "grid" },
 //   title: "Test",
 // };
+
+export const facilityListMeta = {
+  name: 'facility-lists',
```
Comment (pirupius) [inline packages/esm-ugandaemr-app/src/dashboard.meta.tsx:9]: Is this a work in progress cause this isn't being used anywhere

**Review by pirupius — CHANGES_REQUESTED** [review]

File: packages/esm-ugandaemr-app/src/createDashboardLink.tsx (hunk 1 above)
Comment (jabahum) [reply packages/esm-ugandaemr-app/src/createDashboardLink.tsx:10]: this was removed in subsequent prs

File: packages/esm-ugandaemr-app/src/dashboard.meta.tsx (hunk 2 above)
Comment (jabahum) [reply packages/esm-ugandaemr-app/src/dashboard.meta.tsx:9]: its work in progress

**Pull request closed without merging**